libs/ts-rest/core/src/lib/client.ts

```diff
@@ -91,24 +98,20 @@ export type ApiRouteResponse<T> =
       body: unknown;
     };
 
-export type ResponseForRoute<T extends AppRoute> = ApiRouteResponse<
+/**
+ * @deprecated Only safe to use on the client-side. Use `InferResponsesForServer`/`InferResponsesForClient` instead.
+ */
+export type ApiResponseForRoute<T extends AppRoute> = ApiRouteResponse<
```

Comment:
@michaelangrivera `ApiResponseForRoute` was the old name of the type used by `getRouteResponses` previously so I've renamed it back to that. This should be fine since the last merged commits have still not being released.

Reply:
LGTM!